### ISSUE TITLE
README should clarify to install "Go" at the moment.

### ISSUE THREAD

**Issue by JaheimPrice** [TeisybeLT/teisybes_blog#8]
Just for clarity, instruction 2 won't automatically work without Go installed.  

It can be resolved by going to https://go.dev/dl/, and for Windows system there's an an installer. At the moment, since I am using Windows, I used go1.21.4.windows-amd64.msi.

![image](https://github.com/TeisybeLT/teisybes_blog/assets/32816373/49c006bb-ee06-42c9-bd57-ce9898c3a74d)

I assume this is necessary because the go.mod file.
![image](https://github.com/TeisybeLT/teisybes_blog/assets/32816373/b5219fd0-9c00-4dda-830a-5735542b5553)

**Comment by JaheimPrice** [comment]
### POSSIBLE WINDOW ISSUES
Also I tried windows hugo_0.120.4_windows-amd64.zip, but it's is incompatible with hugo theme night fall. 

Using hugo_extended_0.119.0_windows-amd64 instead resolved the explicit incompatibility for me, but still getting "ERROR render of <page> failed:" with page, section, and taxonomy when doing "hugo server" command.

### Extra configuration
For me what fixed it is putting (I'll put as separate issue)
![image](https://github.com/TeisybeLT/teisybes_blog/assets/32816373/b17f4888-f571-4cb3-9a4f-4f46e588ecae)

**Comment by TeisybeLT** [comment]
Yup, 100% correct on that, I forgot about Windows needing to install go manually, will add this to readme soon.
